pkg/metrics/scheduler.go

```diff
@@ -86,7 +86,7 @@ func InitSchedulerMetrics() *SchedulerMetrics {
 			Namespace: Namespace,
 			Subsystem: SchedulerSubsystem,
 			Name:      "application_total",
-			Help:      "Total number of applications. State of the application includes `running` and `completed`.",
+			Help:      "Total number of applications. State of the application includes `running`, `completed` and `failed`.",
```

Comment:
I think the state can also be either 'accepted' or 'rejected'[1], and we can also include these states in the message!

[1] [metrics/init.go#78](https://github.com/apache/yunikorn-core/blob/master/pkg/metrics/init.go#L78C38-L78C38)

Reply:
Correct me if I'm wrong, but I didn't see those labels (i.e. accepted/rejected) in https://github.com/apache/yunikorn-core/blob/1540a5508db1816f11645ab86781a4685de08ebc/pkg/metrics/scheduler.go#L267-L317. Not sure where are these two labels implemented ?

Reply:
These two labels are implemented in [here](https://github.com/apache/yunikorn-core/blob/master/pkg/metrics/scheduler.go#L251-L265)

Reply:
Sorry I missed the part about application_submission_total in metrics/scheduler.go.